### ISSUE TITLE
Deprecating WebSQL

### ISSUE THREAD

**Issue by daleharvey** [pouchdb/pouchdb#6943]
So happy days, IndexedDB is supported in all major browsers (https://caniuse.com/#search=Indexeddb) and once https://github.com/pouchdb/pouchdb/pull/6938 lands we will be using it everywhere, meaning webSQL will become legacy code

I suggest we do a 6.4.1 release that purely 6.4.0 + https://github.com/pouchdb/pouchdb/pull/6938, users who are worried about migrations can then use this release, detect if they have existing webSQL installations and migrate those over to indexedDB

Then at that point, 7.0 can be webSQL free, we can split it out into a new repo as an optional adapter for anyone to use, just not maintained + tested as part of core

**Comment by nolanlawson** [comment]
https://github.com/pouchdb/pouchdb/pull/6938 is merged

**Comment by nolanlawson** [comment]
There's probably no need to split it out into a new repo. Even if we remove it as a dep, running the publish scripts will continue publishing it as an optional dependency, so anyone who wants to use it can. Plus removing all the tangled `websql-core` / `node-websql` deps is probably not worth the trouble.

This plan sounds solid to me!

**Comment by daleharvey** [comment]
Yeh so while ultimately I would prefer them in a seperate repo, I think thats a bit of a big change to make suddenly, can keep them here for now and close this out, after a couple of release / months can take a look at moving them. Thanks :)

**Comment by moses** [comment]
Hi folks: Since the SQLite Plugin depends on the WebSQL adapter, will native SQLite support be deprecated as well within PouchDB?

Thanks for all that you do!
